core/dslmcode/profiles/elmsmedia-7.x-1.x/modules/features/elmsmedia_feature/elmsmedia_feature.views_default.inc

```diff
@@ -495,22 +521,14 @@ function elmsmedia_feature_views_default_views() {
     'image_style' => 'image_card_medium',
     'image_link' => '',
   );
-  /* Field: Content: Images */
+  /* Field: Broken/missing handler */
   $handler->display->display_options['fields']['field_images']['id'] = 'field_images';
   $handler->display->display_options['fields']['field_images']['table'] = 'field_data_field_images';
   $handler->display->display_options['fields']['field_images']['field'] = 'field_images';
   $handler->display->display_options['fields']['field_images']['label'] = '';
   $handler->display->display_options['fields']['field_images']['element_label_colon'] = FALSE;
   $handler->display->display_options['fields']['field_images']['hide_empty'] = TRUE;
   $handler->display->display_options['fields']['field_images']['empty_zero'] = TRUE;
-  $handler->display->display_options['fields']['field_images']['type'] = 'entityreference_entity_view';
```

Comment:
Try exporting elmsmedia_feature again after making sure all features are reverted. It looks like this is currently caliming that field_images is broken / using something that doens't exist so it's droping that field from display for whatever reason